.github/workflows/build-and-deploy-docker-images.yaml

```
@@ -0,0 +1,61 @@
+name: Build and push docker images hosted on everest-ci
+
+on:
+  workflow_dispatch:
+    inputs:
+      force_build:
+        description: 'Force build even if no changes are detected'
+        required: false
+        type: boolean
+  push:
+    branches:
+      - '**'
+    tags:
+      - 'v*'
+
+env:
+  CONTAINER_REGISTRY: ghcr.io
+  CUSTOM_BUILD_ACTION_VERSION: ${{ github.sha }} # Use github.sha to sync with workflow version
+
+jobs:
+  build-and-push-all-images:
+    name: Build and push all docker images
+    strategy:
+      fail-fast: false
+      matrix:
+        include: 
+          - build_args: RELEASE_REPO=${{ github.repository }}
+            directory: docker/images/everest-clang-format
+            image_name: ${{ github.repository_owner }}/everest-clang-format
```

```suggestion
            image_name: ${{ github.repository_owner }}/run-clang-format
```

As the `ENTRYPOINT` directly uses this kind-of-famous script.

---

Right now we already have an image called `everest-clang-format`. But I agree we could discuss renaming it, furthermore discuss naming strategy for our docker images

---

Yes let's keep it everest-* for now and discuss naming more generally